src/EventSubscriber/ContentBlockSubscriber.php

```diff
@@ -11,6 +11,8 @@ class ContentBlockSubscriber implements EventSubscriberInterface
     /** @var \Drupal\wmcontent\WmContentManager */
     private $manager;
 
+    private $updatedHosts = [];
```

Comment:
Added this to make sure we don't save the parent host N times ( where N is the amount of contentblocks attached )